docs/reference/monitoring.txt

```diff
@@ -94,6 +94,12 @@ Sample output:
   D, [2018-09-23T13:47:31.258020 #4692] DEBUG -- : COMMAND | 127.0.0.1:27027 | test.hello | STARTED | {"hello"=>1, "$readPreference"=>{"mode"=>"primary"}, "lsid"=>{"id"=><BSON::Binary:0x47111693353080 type=uuid data=0x730341e880dc40a2...>}}
   D, [2018-09-23T13:47:31.259145 #4692] DEBUG -- : COMMAND | 127.0.0.1:27027 | test.hello | SUCCEEDED | 0.000791175s
 
+.. note::
+
+  The default inspection of a command which is a BSON document gets truncated
```

Comment:
I would add this as a comment inside the `started` method, where we actually use `event.command`. But here is fine, too.

Reply:
Sounds like a better idea, I'll throw that up tomorrow

Reply:
I think this was my first impression too, `event` is not available in the suggested scope?